extension.js

```diff
@@ -245,9 +247,25 @@ class ClipboardIndicator extends PanelMenu.Button {
     this.menu.actor.connect('key-press-event', (_, event) =>
       this._handleGlobalKeyEvent(event),
     );
+    
+    this._settingsChangedId = this.settings.connect(
```

Comment:
Why are these changes necessary? I believe they'll cause a crash if fields like `entries` haven't been initialized yet.

Reply:
Yes but isn't the point of the isDestroying variable to avoid calling that code? Why do we need both?

Reply:
Huh, well that's annoying. Do you know why? To me that means this code isn't actually related because I don't see how it could possible be run if you have the destroying flag is set.

I'd really like to know why the flag doesn't work, but we can also flip the problem around. Add an `_isLoaded` variable that we set to true at the end of `buildClipboardStateFromLog` and then check for it in each of the callbacks. `_setupSelectionChangeListener` will be annoying cuz you'll have to add the if check inside the relevant callbacks initialized by that method.